package.json

```diff
@@ -4,87 +4,85 @@
   "description": "Use Amazon Athena with Grafana",
   "scripts": {
     "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
-    "dev": "webpack -c ./.config/webpack/webpack.config.ts --env development",
-    "e2e": "yarn cypress install && yarn grafana-e2e run",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "e2e": "yarn exec cypress install && yarn exec grafana-e2e run",
```

Comment:
just curious what does adding exec do? Is this just a new syntax for a new version of yarn or something?

Reply:
not really sure. it was added by `create-plugin` when i ran the update command. but it's available in yarn classic and when i ran `yarn e2e` locally, it still seemed to run the tests fine.